Update yes :)

Nullacron opened the pull request:
Added, removed, edited, etc.

Nullacron committed:
Update README.md

Nullacron committed:
As per your request, lord GoingCrazyDude.

Nullacron committed:
Forgot some credits.

Nullacron committed:
Fixing... stuff....

Nullacron committed:
Changing more stuff

Nullacron committed:
Changing A titling

Nullacron committed:
More

Nullacron committed:
Update README.md

Nullacron committed:
Update README.md

Nullacron committed:
Update README.md

Nullacron committed:
Update README.md

ShadowCodeX-debug commented:
Did things yes. :)

Nullacron committed:
Update README.md

Nullacron committed:
Update README.md

Nullacron committed:
Update README.md

Nullacron committed:
Update README.md

GoingCrazyDude commented on README.md:
````diff
@@ -1,339 +1,282 @@
-```
+# Rendering and Graphics
 
-### Change Rendering Mode
````
Keep the rendering mode section the same no need to change it

GoingCrazyDude commented on README.md:
```diff
@@ -1,330 +1,307 @@
-# NO MORE CLOSET CHEATERS
-![drake](https://github.com/pizzaboxer/bloxstrap/assets/67791946/303dfde5-1968-435e-8660-249e8b6001c6)
+> [!NOTE]
+> Local Fast Flags are being patched out. Expect your flags to not work soon enough.
```
This should be "Local ClientSettings loading is being removed. Expect FastFlags to not be modifiable by the client soon."

ShadowCodeX-debug replied:
The word FastFlags are more known than ClientSettings. I will change it a little bit though.

GoingCrazyDude commented on README.md:
````diff
@@ -1,330 +1,307 @@
-```
+# Change Rendering Mode
 
-### Change Rendering Mode
````
Keep the "Change Rendering Mode" section same as the current one in the official repo.

ShadowCodeX-debug replied:
I already did this.

Nullacron committed:
Update README.md

Nullacron committed:
Update README.md

Nullacron committed:
!NOTE > !IMPORTANT

Nullacron committed:
Update patchnotedormate.md

Nullacron committed:
Update patchnotedormate.md

Nullacron committed:
Delete patchnotedormate.md

ShadowCodeX-debug commented:
Did more things yes? :)

ShadowCodeX-debug commented:
Hey @GoingCrazyDude which one looks better?

```json
{ "NewLookConcept": true }
```

Or just keep the original?

```json
{
  "OldLook": true
}

ShadowCodeX-debug commented:
(It's for ones that are lone flags.

Nullacron committed:
Code Block Format Changes

- Made single FastFlags single line codeblocks
- Replaced tab spaces with double spaces.

GoingCrazyDude commented:
I dont like the changes. Going to update it myself.

ShadowCodeX-debug commented:
@GoingCrazyDude Y’know if you don’t like my edits, don’t use them. (If this was code I would call you a skid.)

GoingCrazyDude commented:
> @GoingCrazyDude Y’know if you don’t like my edits, don’t use them. (If this was code I would call you a skid.)

I haven't copied any of your edits idk what you're talking about
You can't call me a skid when I'm compiling publicly available information

ShadowCodeX-debug commented:
You defiantly copied the uncap fps.
